### ISSUE TITLE
Polls 6.1.1 - Error 404 poll not found for public polls 

### ISSUE THREAD

**Issue by nicovd737** [nextcloud/polls#3334]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### What went wrong, what did you observe?

We create a new poll and open it to public share via link or QR code.
When external persons try to access the poll, they have a 404 - poll not found displayed.

### What did you expect, how polls should behave instead?

Public polls must be available and visible to public when public option is set

### What steps does it need to replay this bug?

1. Create a poll
2. Set it to public
3. try to access via the public link or QR code

### Installation method

Installed/updated from the appstore (Apps section of your site)

### Installation type

Updated from a minor version within same major version (i.e. 4.0.0 to 4.1.1)

### Affected polls version

6.1.1

### Which browser did you use, when experiencing the bug?

- [X] Firefox
- [X] Chrome
- [X] Chromium/Chromium based (i.e. Edge)
- [X] Safari
- [X] Other/Don't know

### Add your browser log here

```shell
Will add a printscreen
```

### NC version

Other/Don't know

### Other Nextcloud version

28.0.2

### PHP engine version

PHP 8.1

### Database engine

MariaDB

### Database Engine version or other Database

10.6

### Which user-backends are you using?

- [X] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other/Don't know

### Add your nextcloud server log here

```shell
{"reqId":"AHqgiuvno42p7evvZaec","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/share?time=1708584640803","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"AHqgiuvno42p7evvZaec","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/share?time=1708584640803","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"bn27WACqIduNDpqOPMJ1","level":2,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/options?time=1708584640950","message":"Ignoring requested pollId . Keeping share pollId of share(nXok93AZ): 185","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"bn27WACqIduNDpqOPMJ1","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/options?time=1708584640950","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"bn27WACqIduNDpqOPMJ1","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/options?time=1708584640950","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"0tYUh8I5bkAokXTneC4X","level":2,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/subscription","message":"Ignoring requested pollId . Keeping share pollId of share(nXok93AZ): 185","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"bn27WACqIduNDpqOPMJ1","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/options?time=1708584640950","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"0tYUh8I5bkAokXTneC4X","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/subscription","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}{"reqId":"bn27WACqIduNDpqOPMJ1","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/options?time=1708584640950","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"k24Axuwr7A2QZfQc8zEk","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/share?time=1708584640953","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"bn27WACqIduNDpqOPMJ1","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/options?time=1708584640950","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"bn27WACqIduNDpqOPMJ1","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/options?time=1708584640950","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"bn27WACqIduNDpqOPMJ1","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/options?time=1708584640950","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"k24Axuwr7A2QZfQc8zEk","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/share?time=1708584640953","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"Jwy6VZMqUZrUOuuvbf6M","level":2,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/comments?time=1708584640950","message":"Ignoring requested pollId . Keeping share pollId of share(nXok93AZ): 185","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"Loc8EnUW4LRfrXb8aGmI","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/poll?time=1708584640948","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"Jwy6VZMqUZrUOuuvbf6M","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/comments?time=1708584640950","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"Loc8EnUW4LRfrXb8aGmI","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"user_ldap","method":"GET","url":"/apps/polls/s/nXok93AZ/poll?time=1708584640948","message":"Calling LDAP function ldap_explode_dn with parameters [\"Nicolas.Bour\",0]","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"user_ldap"}}
{"reqId":"Zbm15EaMNqf6UgqFB2xb","level":2,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/votes?time=1708584640951","message":"Ignoring requested pollId . Keeping share pollId of share(nXok93AZ): 185","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"Zbm15EaMNqf6UgqFB2xb","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/votes?time=1708584640951","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"Loc8EnUW4LRfrXb8aGmI","level":0,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"polls","method":"GET","url":"/apps/polls/s/nXok93AZ/poll?time=1708584640948","message":"no user found, returned fake user","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","data":{"app":"polls"}}
{"reqId":"Zbm15EaMNqf6UgqFB2xb","level":3,"time":"2024-02-22T08:50:39+02:00","remoteAddr":"136.23.1.113","user":"--","app":"index","method":"GET","url":"/apps/polls/s/nXok93AZ/votes?time=1708584640951","message":"OCA\\Polls\\Db\\VoteMapper::findByPollAndUser(): Argument #1 ($pollId) must be of type int, null given, called in /var/www/html/custom_apps/polls/lib/Service/VoteService.php on line 65 in file '/var/www/html/custom_apps/polls/lib/Db/VoteMapper.php' line 93","userAgent":"Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Mobile Safari/537.36","version":"28.0.2.5","exception":{"Exception":"Exception","Message":"OCA\\Polls\\Db\\VoteMapper::findByPollAndUser(): Argument #1 ($pollId) must be of type int, null given, called in /var/www/html/custom_apps/polls/lib/Service/VoteService.php on line 65 in file '/var/www/html/custom_apps/polls/lib/Db/VoteMapper.php' line 93","Code":0,"Trace":[{"file":"/var/www/html/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/html/lib/base.php","line":1069,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/html/index.php","line":39,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","Line":169,"Previous":{"Exception":"TypeError","Message":"OCA\\Polls\\Db\\VoteMapper::findByPollAndUser(): Argument #1 ($pollId) must be of type int, null given, called in /var/www/html/custom_apps/polls/lib/Service/VoteService.php on line 65","Code":0,"Trace":[{"file":"/var/www/html/custom_apps/polls/lib/Service/VoteService.php","line":65,"function":"findByPollAndUser","class":"OCA\\Polls\\Db\\VoteMapper","type":"->"},{"file":"/var/www/html/custom_apps/polls/lib/Controller/PublicController.php","line":130,"function":"list","class":"OCA\\Polls\\Service\\VoteService","type":"->"},{"file":"/var/www/html/custom_apps/polls/lib/Controller/BasePublicController.php","line":61,"function":"OCA\\Polls\\Controller\\{closure}","class":"OCA\\Polls\\Controller\\PublicController","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/html/custom_apps/polls/lib/Controller/PublicController.php","line":129,"function":"response","class":"OCA\\Polls\\Controller\\BasePublicController","type":"->"},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":230,"function":"getVotes","class":"OCA\\Polls\\Controller\\PublicController","type":"->"},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":137,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":184,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/html/lib/private/Route/Router.php","line":315,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/html/lib/base.php","line":1069,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/html/index.php","line":39,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/html/custom_apps/polls/lib/Db/VoteMapper.php","Line":93},"message":"OCA\\Polls\\Db\\VoteMapper::findByPollAndUser(): Argument #1 ($pollId) must be of type int, null given, called in /var/www/html/custom_apps/polls/lib/Service/VoteService.php on line 65 in file '/var/www/html/custom_apps/polls/lib/Db/VoteMapper.php' line 93","exception":{},"CustomMessage":"OCA\\Polls\\Db\\VoteMapper::findByPollAndUser(): Argument #1 ($pollId) must be of type int, null given, called in /var/www/html/custom_apps/polls/lib/Service/VoteService.php on line 65 in file '/var/www/html/custom_apps/polls/lib/Db/VoteMapper.php' line 93"}}
```

### Additional environment informations

![image](https://github.com/nextcloud/polls/assets/28760546/eb5cdb7c-006f-471c-8b00-7054231be0fb)

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "xxx.xxxx.xxxxx"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "28.0.2.5",
        "overwriteprotocol": "https",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "3306",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "NTLM",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "10587",
        "default_phone_region": "CH",
        "filelocking.enabled": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "password": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/xxx.xxx.xxx",
        "htaccess.RewriteBase": "\/",
        "maintenance": false,
        "theme": "",
        "loglevel": 0,
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "updater.release.channel": "stable",
        "twofactor_enforced": "false",
        "twofactor_enforced_groups": [
            "xxx",
            "xxx"
        ],
        "twofactor_enforced_excluded_groups": [],
        "mysql.utf8mb4": true,
        "app_install_overwrite": [
            "spreed",
            "twofactor_rcdevsopenotp",
            "occweb",
            "mindmap_app",
            "drawio",
            "officeonline",
            "files_mindmap",
            "openotp_auth",
            "impersonate",
            "integration_moodle",
            "talk_matterbridge",
            "files_downloadactivity",
            "imageconverter",
            "emlviewer"
        ],
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "data-fingerprint": "550994713cc295254572150adaf2b32c",
        "logtimezone": "CEST",
        "mail_sendmailmode": "smtp",
        "memories.exiftool": "\/var\/www\/html\/custom_apps\/memories\/bin-ext\/exiftool-amd64-glibc",
        "memories.vod.path": "\/var\/www\/html\/custom_apps\/memories\/bin-ext\/go-vod-amd64",
        "simpleSignUpLink.shown": false
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.20.0
  - admin_audit: 1.18.0
  - announcementcenter: 6.7.0
  - audioplayer: 3.4.1
  - bruteforcesettings: 2.8.0
  - cadviewer: 9.44.2
  - calendar: 4.6.5
  - circles: 28.0.0-dev
  - cloud_federation_api: 1.11.0
  - comments: 1.18.0
  - contacts: 5.5.2
  - contactsinteraction: 1.9.0
  - dashboard: 7.8.0
  - dav: 1.29.1
  - deck: 1.12.2
  - drawio: 3.0.2
  - external: 5.3.1
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_accesscontrol: 1.18.0
  - files_automatedtagging: 1.18.0
  - files_external: 1.20.0
  - files_fulltextsearch: 28.0.0
  - files_mindmap: 0.0.30
  - files_pdfviewer: 2.9.0
  - files_reminders: 1.1.0
  - files_retention: 1.17.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - firstrunwizard: 2.17.0
  - forms: 4.1.1
  - geoblocker: 0.5.13
  - groupfolders: 16.0.3
  - imageconverter: 1.3.5
  - impersonate: 1.15.0
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - mail: 3.5.6
  - memories: 6.2.2
  - nextcloud_announcements: 1.17.0
  - notes: 4.9.2
  - notifications: 2.16.0
  - oauth2: 1.16.3
  - officeonline: 2.1.0
  - openotp_auth: 1.5.1
  - passman: 2.4.8
  - password_policy: 1.18.0
  - photos: 2.4.0
  - polls: 6.1.1
  - privacy: 1.12.0
  - provisioning_api: 1.18.0
  - recommendations: 2.0.0
  - related_resources: 1.3.0
  - serverinfo: 1.18.0
  - settings: 1.10.1
  - sharebymail: 1.18.0
  - side_menu: 3.11.8
  - smb_test: 0.3.8
  - spreed: 18.0.3
  - survey_client: 1.16.0
  - systemtags: 1.18.0
  - talk_matterbridge: 1.26.0
  - tasks: 0.15.0
  - text: 3.9.1
  - theming: 2.3.0
  - theming_customcss: 1.15.0
  - twofactor_backupcodes: 1.17.0
  - twofactor_nextcloud_notification: 3.8.0
  - updatenotification: 1.18.0
  - user_ldap: 1.19.0
  - user_status: 1.8.1
  - viewer: 2.2.0
  - weather_status: 1.8.0
  - workflowengine: 2.10.0
Disabled:
  - emlviewer: 1.0.9 (installed 1.0.9)
  - encryption: 2.16.0 (installed 2.10.0)
  - extract: 1.3.6 (installed 1.3.6)
  - files_downloadactivity: 1.16.0 (installed 1.16.0)
  - files_rightclick: 0.15.1 (installed 1.6.0)
  - fulltextsearch: 28.0.0 (installed 28.0.0)
  - fulltextsearch_elasticsearch: 28.0.0 (installed 28.0.0)
  - integration_moodle: 1.0.2 (installed 1.0.2)
  - ransomware_protection: 1.14.0 (installed 1.14.0)
  - support: 1.11.0 (installed 1.5.0)
  - suspicious_login: 6.0.0
  - twofactor_totp: 10.0.0-beta.2 (installed 6.4.0)
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Additional Information

Happens only after poll update.

**Comment by dartcafe** [comment]
see #3326 and #3329.

fixed in [6.1.3](https://github.com/nextcloud/polls/issues/3332)

**Comment by nicovd737** [comment]
Hello,

Someone yesterday told me to open a new issue ... that's what i did ...

**Comment by nicovd737** [comment]
Please re-open. I have installed your update, but still having issue 404 poll not found

**Comment by dartcafe** [comment]
Yep. You are right. I read your description too fast, but even with externjal users, I  could not replay the error until I set the "hide results" setting.
